server/public/model/feature_flags.go

```diff
@@ -93,7 +93,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.NormalizeLdapDNs = false
 	f.UseCaseOnboarding = true
 	f.GraphQL = false
-	f.InsightsEnabled = true
+	f.InsightsEnabled = false
```

Comment:
@harshilsharma63 this would set any new server defaults to `false` but existing servers that have it enabled with have it still set to `true`. I believe we may need to do something else for the case of existing servers

Reply:
Discussed this a bit more offline and @harshilsharma63 pointed out there is nothing more to do here for existing servers as the feature flags are not stored in the config. Thanks @harshilsharma63!

Reply:
Yes, correct. Feature flag is the only way we can enable or disable this feature. There is no other system console or config setting.